packages/webpack/src/loader.ts

```diff
@@ -103,14 +104,16 @@ export default async function vocabLoader(this: LoaderContext) {
 
   const loadedLanguages = Object.keys(loadedTranslation.languages);
 
+  const exportName = findExportNames(source)[0];
   const result = /* ts */ `
     import { createLanguage, createTranslationFile } from '@vocab/webpack/${target}';
 
-    export default createTranslationFile({
+    const translation = createTranslationFile({
```

Comment:
This seems to be a change to the generated code, but doesn't seem related to the PR description. Why this change?

Reply:
Explained in this thread https://github.com/seek-oss/vocab/pull/135#discussion_r1166289149